components/home/sections/CreateCollective.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+
+import { Box, Flex } from '../../Grid';
+import Link from '../../Link';
+import StyledButton from '../../StyledButton';
+import { H4 } from '../../Text';
+
+const Wrapper = styled(Box)`
+  background-image: ${props =>
+    props.hovering
+      ? "url('/static/images/home/create-collective-bg-illustration-hover-sm.png')"
+      : "url('/static/images/home/create-collective-bg-illustration-sm.png')"};
```

Comment:
That shouldn't be done this way, because it creates a flickering issue while the image first load. In `Contribute to the platform` from https://github.com/opencollective/opencollective/issues/3321, I described another way to do that that prevents this flickering issue.